src/slurm_plugin/clustermgtd.py

```diff
@@ -150,6 +150,7 @@ class ClustermgtdConfig:
         "terminate_drain_nodes": True,
         "terminate_down_nodes": True,
         "orphaned_instance_timeout": 300,
+        "ec2_backing_instance_timeout": 300,
```

Comment:
I think the 5 minutes default is too much. Let's sync on this

Reply:
I'm ok with shortening to say 2 mins to allow 2-3 loops